app/assets/stylesheets/components/_markdown-editor.scss

```diff
@@ -79,5 +79,36 @@
 
   .govuk-textarea {
     overflow: scroll;
+    padding-top: govuk-spacing(8);
   }
 }
+
+.app-c-markdown-editor-toolbar__icon {
+  display: inline-block;
```

Comment:
This should be `display:block;` and `.app-c-markdown-editor-toolbar__button` should be `inline-block;`. This way you won't have alignment issues and the focus will be rectangular.

Reply:
The buttons should follow the focus pattern we have with other components. You can achieve that with `@include govuk-focusable;` in `.app-c-markdown-editor-toolbar__button`

Reply:
Hi alex, this has all been done.